### ISSUE TITLE
[A11yMAS][Accessibility][High Contrast][MAS4.3.1][Eclipse][Wipro]:Expand/collapse button and Version text are clearly not visible for selected pack in "Available Software" window in both highcontrast modes

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1425]
**Repro Steps:**

1. Open and Start Eclipse.
2. Press "Alt+H" to open Help menu.
3. Navigate and select Install new software Menu item under Help menu.
4. In Available Software dialog , navigate to Work with text edit box and provide http://dl.microsoft.com/eclipse/ link and hit enter
5. Then tab to available packs(eg; Azure Tool kit for Java) which is present under Name and observe the expand/collapse button and version text for the selected pack in both HC modes.

**Expected Result:**
Expand/collapse button and version text for selected pack should be displayed in turquoise color background with black text in HC black mode and Violet color background with white text in HC white mode

**Actual Result:**
Expand/collapse button and version text for selected pack are displayed as turquoise
color background with white text in HC black mode and Violet color background with black text in HC white mode
 
**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version:** Version 1709(Os Build 16299.192)

![35718916-bbe81a98-080c-11e8-90b5-3c2b6f5593bb](https://user-images.githubusercontent.com/22956130/35719265-a709d5d8-080e-11e8-95fc-e3ce5b42259b.jpg)

![selected text under version is not following hc standards in available software window in both hc modes](https://user-images.githubusercontent.com/22956130/35719150-039404be-080e-11e8-8bf6-9b557342230f.jpg)

**Comment by Eskibear** [comment]
It is determined by Eclipse IDE. We cannot fix it.

**Comment by Eskibear** [comment]
The color is as expected in:
```
Eclipse Java EE IDE for Web Developers.

Version: Oxygen.3a Release (4.7.3a)
Build id: 20180405-1200
```
Please help to re-verify.

![image](https://user-images.githubusercontent.com/2351748/43440506-2fde0be8-94ca-11e8-8054-6b376647db4b.png)

**Comment by WiproPSAccTest** [comment]
![screenshot 8](https://user-images.githubusercontent.com/22956130/43780148-8c86e380-9a77-11e8-89c0-af166e111b79.png)
Verified the issue in Version Oxygen 3a Release (4.7.3a) .Issue Still Repros .

**Comment by WiproPSAccTest** [comment]
Verified the issue in Version Oxygen 3a Release (4.7.3a)
OS Version - 1803 (OS Build 17134.167)
 .Issue Still Repro .
![screenshot 1299](https://user-images.githubusercontent.com/22956130/43827555-b945c328-9b17-11e8-8539-e50662f23d82.png)

**Comment by Eskibear** [comment]
* **Importance of experience**: somewhat important. It's about the one-time installation.
* **Quality of experience**: not impacted. A simple workaround is available, users can install the plugin from **Help->Marketplace** instead of **Help->Install New Software**.  
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by umasarath52** [comment]
Discussed with Ryan and approved to close as non-blocking scenario

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749863
